software/simulation.md

```diff
@@ -86,5 +97,34 @@ ros2 launch turtlebot4_ignition_bringup turtlebot4_ignition.launch.py model:=lit
     <figcaption>TurtleBot 4 Lite in Ignition Gazebo</figcaption>
 </figure>
 
+{% endtab %}
+{% tab ignition jazzy %}
+
+The `turtlebot4_gz.launch.py` launch file has several [launch configurations](turtlebot4_simulator.md#ignition-bringup) that allow the user to customize the simulation.
```

Comment:
In all tabs can you update the link for the new heading?

Reply:
Done